src/server/es/sort.js

```diff
@@ -5,7 +5,7 @@ import { UserInputError } from 'apollo-server';
  * @param {object} graphqlSort
  */
 const getESSortBody = (graphqlSort, esInstance, esIndex) => {
```

Comment:
can you add a comment describing the purpose of this function?

Reply:
addressed